src/formats/MMTF.cpp

```diff
@@ -68,6 +68,20 @@ MMTFFormat::MMTFFormat(std::string path, File::Mode mode, File::Compression comp
         auto file = TextFile(std::move(path), mode, compression);
         auto buffer = file.readall();
         decode(buffer.data(), buffer.size(), file.path());
+        if (!mmtf::isDefaultValue(structure_.atomIdList)) {
+            // If ids are not ordered or are missing consecutive values the atoms
+            // have to be re-ordered.
+            bool isValidIdOrder = false;
+            if (structure_.atomIdList[0] == 1) {
+                isValidIdOrder = std::is_sorted(
```

Comment:
is this hacking `std::is_sorted` to check if the vector contains consecutive values?

Reply:
If so it might be worth a comment =)

Reply:
yep, could not think of a better way